src/typescript.ts

```diff
@@ -26,6 +29,8 @@ function hello(_?: string): bigint {
 ['', undefined].forEach((_lib) => {
   // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing,@typescript-eslint/strict-boolean-expressions
   const thing = _lib || {};
+  // this is a comment for the regex below
```

Comment:
@jpolavar reverted back to eslint-config 5.3.0. since this is failing with a `Missing comment for regular expression  @checkdigit/regular-expression-comment`